src/auth/OAUthWithIDTokenValidation.js

```diff
@@ -0,0 +1,92 @@
+var jwt = require('jsonwebtoken');
+var jwksClient = require('jwks-rsa');
+var Promise = require('bluebird');
+
+var ArgumentError = require('rest-facade').ArgumentError;
+
+/**
+ * @class
+ * Abstracts the `oauth.create` method with additional id_token validation\
+ * @constructor
+ * @memberOf module:auth
+ *
+ * @param  {Object}              options                Authenticator options.
+ * @param  {String}              options.baseUrl        The auth0 account URL.
```

Comment:
"Auth0"

Reply:
Also ... is this specific option used anywhere?

Reply:
removed the param